de.metas.contracts/src/test/java/de/metas/contracts/impl/ContractChangeBLTest.java

```diff
@@ -131,6 +133,31 @@ public void cancel_a_Contract_which_was_extended_using_a_date_from_extended_cont
 		assertSubscriptionProgress(extendedContract, 3);
 		assertThat(contract.getMasterEndDate()).isEqualTo(extendedContract.getMasterEndDate());
 	}
+	
+	@Test
+	public void closeContract_test()
+	{
```

Comment:
* this test seems to be about *not* closing the term, right? if so, pls update the method name
* and imho you can remote the `_test`, it's redundant